addons/resource.language.en_gb/resources/strings.po

```diff
@@ -13836,12 +13845,12 @@ msgstr ""
 
 #: xbmc/peripherals/devices/peripheralcecadapter.cpp
 msgctxt "#36011"
-msgid "Could not detect the CEC com port. Set it up manually."
+msgid "Couldn't detect the CEC com port. Set it up manually."
```

Comment:
Since the next message uses "Please blablabla" I belive the latter sentence here should be `Please configure it manually.`.